client/components/mma/identity/publicProfile/PublicProfile.tsx

```diff
@@ -63,7 +60,9 @@ export const PublicProfile = (_: { path?: string }) => {
 	const usernameDisplay = (u: User) => (
 		<>
 			<WithStandardTopMargin>
-				<PageSection title="Username">{u.username}</PageSection>
+				<PageSection title="Username">
+					<span data-cy="username-display">{u.username}</span>
```

Comment:
just curious why do we need this span here now? is it just for cypress?

Reply:
Yeah, it's just for Cypress. Makes it much easier to target it.